metrics/src/vespa/metrics/repo.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+#pragma once
+
+#include <vespa/vespalib/stllike/string.h>
+#include <vespa/vespalib/metrics/handle.h>
+
+namespace metrics {
+
+struct MetricNameIdTag {};
+struct DescriptionIdTag {};
+struct TagKeyIdTag {};
+struct TagValueIdTag {};
+
+using MetricNameId = vespalib::metrics::Handle<MetricNameIdTag>;
+using DescriptionId = vespalib::metrics::Handle<DescriptionIdTag>;
+using TagKeyId = vespalib::metrics::Handle<TagKeyIdTag>;
+using TagValueId = vespalib::metrics::Handle<TagValueIdTag>;
+
+struct Repo {
```

Comment:
Our use of the term "repo" is sufficiently overloaded that (especially in a directory that already has a "manager") I think it would be beneficial to rename this to e.g. `NameRepo` or `StringRepo`. Similarly with the file name.